src/openforms/registrations/contrib/objects_api/submission_registration.py

```diff
@@ -511,7 +513,12 @@ def get_update_payment_status_data(
                 variables_registry=variables_registry,
             )
             if variable.key
-            in ["payment_completed", "payment_amount", "payment_public_order_ids"]
+            in [
+                "payment_completed",
+                "payment_amount",
+                "payment_public_order_ids",
+                "provider_payment_ids",
+            ]
```

Comment:
this list of variable names is duplicated in a number of places, can we instead define it as a constant `PAYMENT_VARIABLE_NAMES` and use that so we have a single source of truth?